main.py

```diff
@@ -0,0 +1,5 @@
+#from controller import start
+print('Всем привет, представляем Вашему вниманию наш миленький справочник')
+import controller
+if __name__ == '__main__':
+    controller.start()
```

Comment:
Между импортами должна быть пустая строка
```suggestion
import controller

if __name__ == '__main__':
    controller.start()
```